rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -427,21 +448,74 @@
 
       newContainers.merge(containers)
         .attr("class", "chart-container " + (chartKeys.length === 1 ? 'single-chart' : 'two-charts'))
-        .each(function (index, __, charts) {
-          renderChart(d3.select(charts[index]), index);
-        })
         .selectAll("svg")
         .attr("width", calcChartWidth())
         .attr("height", calcChartHeight())
         .selectAll(".chart")
         .attr("transform", "translate(" + ChartSettings.left + "," + ChartSettings.top + ")")
         .select(".xaxis");
 
+      newContainers.each(function (_, index, charts) {
+        renderChart(d3.select(charts[index]), index);
+      });
+
       // Remove charts if selectedRuns reduces in size.
       // We don't need to do this for all the inner elements, the remove will propagate.
       containers.exit().remove();
     }
 
+    function renderMathHJax() {
+      window.MathJax.Hub.Config({
```

Comment:
Please pass `MathJax` in as an argument to this module rather than accessing it via `window.MathJax`. 

```javascript
(function($, _, MathJax) {
  'use strict';
  //.....
  window.RapidResponseAsideInit = initializeRapidResponseAside;
}($, _, MathJax));
```

Reply:
Done